py/tests/py_image_layer/BUILD.bazel

```diff
@@ -26,3 +26,18 @@ assert_tar_listing(
     actual = [":my_app_layers"],
     expected = ":my_app_layers.listing",
 )
+
+# Case 2: With `strip_prefix` and `root`
+py_image_layer(
+    name = "my_app_layers_strip_prefix",
+    binary = ":my_app_bin",
+    platform = ":linux_amd64",
+    strip_prefix = "py\\/tests\\/py_image_layer\\/my_app_bin",
```

Comment:
it should be possible to pass this string to awk `-v strip_prefix=` so that we don't have to escape slashes.

Reply:
basically pass this information to awk using its `-v strip_prefix=/py/test/my_app_bin` and use `strip_prefix` variable within awk script.